samples/sample_vpp/include/sample_vpp_utils.h

```diff
@@ -437,10 +437,10 @@ class SurfaceVPPStore
 public:
     struct SurfVPPExt
     {
-        SurfVPPExt(mfxFrameSurface1* pSurf = 0, mfxExtVppAuxData* pExt = 0):pSurface(pSurf),pExtVpp(pExt)
+         SurfVPPExt(mfxFrameSurfaceWrap* pSurf = 0, mfxExtVppAuxData* pExt = 0):pSurface(pSurf),pExtVpp(pExt)
```

Comment:
kill the extra space before ```SurfVPPExt``` and use ```nullptr``` for pointer's default value

Reply:
Done